sdc-plugin/tests/get_clocks/get_clocks.tcl

```diff
@@ -0,0 +1,35 @@
+yosys -import
+plugin -i sdc
+plugin -i design_introspection
+# Import the commands from the plugins to the tcl interpreter
+yosys -import
+
+read_verilog $::env(DESIGN_TOP).v
+read_verilog -specify -lib -D_EXPLICIT_CARRY +/xilinx/cells_sim.v
+read_verilog -lib +/xilinx/cells_xtra.v
+hierarchy -check -auto-top
+# Start flow after library reading
+synth_xilinx -vpr -flatten -abc9 -nosrl -nodsp -iopad -run prepare:check
```

Comment:
Drop the `-vpr` flag as we no longer use it.  Actually I recommend we consider testing with just `synth_xilinx` with no flags.

Reply:
Dropped the `-vpr` flag